interface-definitions/vpn_ipsec.xml.in

```diff
@@ -1087,6 +1100,20 @@
                       </node>
                     </children>
                   </tagNode>
+                  <leafNode name="virtual-address">
```

Comment:
VRRP uses only `address` maybe we should stick to this?

Reply:
I think `virtual-address` is more clear for the user as it related option of swanctl configuration `connections.<conn>.vips = x.x.x.x, z.z.z.z`
It requests getting an address from a remote site that will be attached to some interface after establishing an IPSec session.
@zdc What do you think?

Reply:
Also, it will be more unclear if we'll do this migration [T4118](https://phabricator.vyos.net/T4118)